packages/account/src/Components/form-field-info/form-field-info.tsx

```diff
@@ -0,0 +1,47 @@
+import React from 'react';
+import { Popover } from '@deriv/components';
+import { TPopoverProps } from '@deriv/components/src/components/types';
+import { isMobile } from '@deriv/shared';
+
+/**
+ * A component that renders a popover with an info icon.
+ *
+ * @param {TPopoverProps} props - Props for the popover component.
+ * @returns React.ReactElement - A React component.
+ */
+export const FormFieldInfo = (props: Omit<TPopoverProps, 'alignment'>) => {
+    const [is_open, setIsOpen] = React.useState(false);
+    const ref = React.useRef<HTMLDivElement>(null);
+
+    React.useEffect(() => {
+        // Close popover when clicking outside of it
+        const handleClickOutside = (event: MouseEvent) => {
```

Comment:
Can we move this out to hooks package so other devs can use this without creating it so many times. Something like this. If this is out of scope feel free to ignore my comment 🙏 

```suggestion
        import { useEffect, useRef } from 'react';

const useClickOutside = (callback) => {
  const ref = useRef(null);

  useEffect(() => {
    const handleClickOutside = (event) => {
      if (ref.current && !ref.current.contains(event.target)) {
        callback();
      }
    };

    document.addEventListener('mousedown', handleClickOutside);

    return () => {
      document.removeEventListener('mousedown', handleClickOutside);
    };
  }, [callback]);

  return ref;
};

export default useClickOutside;
```

Reply:
LGBTQ